docs/documentation.html

```diff
@@ -69,15 +69,15 @@ <h2><a id="security" href="#security">7. Security</a></h2>
     <h2><a id="connect" href="#connect">8. Kafka Connect</a></h2>
     <!--#include virtual="connect.html" -->
 
-    <h2><a id="streams" href="/11/documentation/streams">9. Kafka Streams</a></h2>
+    <h2><a id="streams" href="/streams">9. Kafka Streams</a></h2>
```

Comment:
Will this be leading to `https://kafka.apache.org/streams/` which is a 404?

Reply:
Think you are right -- I miss-understood the layout/nesting... Will update this.